src/python/grpcio_tests/tests/interop/methods.py

```diff
@@ -78,11 +101,12 @@ def StreamingInputCall(self, request_iterator, context):
         aggregated_payload_size=aggregate_size)
 
   def FullDuplexCall(self, request_iterator, context):
+    self.MaybeEchoMetaData(context)
     for request in request_iterator:
-      if request.HasField('response_status'):
-        context.set_code(request.response_status.code)
-        context.set_details(request.response_status.message)
+      self.EchoStatusAndMessage(request, context)
+      print request.response_parameters
```

Comment:
Leftover debugging printing? Or is this emission part of the test implementation?